Buildasaur/ServiceAuthentication.swift

```diff
@@ -66,6 +66,8 @@ class ServiceAuthenticator {
         switch service {
         case .GitHub:
             return self.getGitHubParameters()
+        case .EnterpriseGitHub:
```

Comment:
Remove this and let it assert if Enterprise gets asked for params, OAuth won't be allowed for now